src/utils/method-helpers.js

```diff
@@ -28,3 +29,6 @@ const sleep = ms => new Promise(resolve => setTimeout(resolve, ms));
 
 export const delay = (delayed, ms) =>
   Promise.all([delayed, sleep(ms)]).then(([data]) => data);
+
+export const translate = (key, lang, interpolation = null) =>
```

Comment:
Can you avoid passing the parameter of the current language (`lang`)? Calculate the language that the user chose or which is specified in the settings and use it?